tests/spread/plugins/v2/snaps/colcon-ros2-foxy-rlcpp-hello/snap/snapcraft.yaml

```diff
@@ -0,0 +1,22 @@
+name: colcon-ros2-foxy-rlcpp-hello
+version: "1.0"
+summary: hello world
+description: |
+  A ROS2 rlcpp-based workspace.
+
+grade: stable
+confinement: strict
+base: core20
+
+apps:
+  colcon-ros2-foxy-rlcpp-hello:
+    command: opt/ros/foxy/bin/ros2 run colcon_ros2_rlcpp_hello colcon_ros2_rlcpp_hello
+    plugs: [network, network-bind]
+    extensions: [ros2-foxy]
+
+parts:
+  colcon-part:
+    plugin: colcon
+    source: .
+    build-packages: []
```

Comment:
You probably want g++ and make here.

Reply:
Good point, will do. Interesting how that was never a problem for me, must be all that destructive mode :P